test/unit/tasks/data_hygiene/attachment_attribute_updater_test.rb

```diff
@@ -0,0 +1,66 @@
+require "test_helper"
+
+class AttachmentAttributeUpdaterTest < ActiveSupport::TestCase
+  extend Minitest::Spec::DSL
+
+  describe DataHygiene::AttachmentAttributeUpdater do
+    let(:attachment) { build(:file_attachment, attachment_data: build(:attachment_data), command_paper_number: original_command_paper_number) }
+    let(:original_command_paper_number) { "CM123" }
```

Comment:
Minor: in general it can be confusing to have a top-level `let` that gets overridden further down. In this case, we could move it into the individual `context`s, and the first `it` for a real run.